### ISSUE TITLE
Provide A11Y Audit AI Skill for Teams

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4560]
### Scope

Because AI Agents become more and more mature, they are now able to conducts professional automated Accessibility Audits with just one prompt. To support product teams, the PDS should provide a Skill file (for different AI Platforms like Copilot, Cursor and Claude) for download/install and dedicated docs how to conduct an Accessibility Audit with an AI Agent.

### Notes

**Evaluate what to integrate in such a Skill, e.g.:**

- Which tools to use?
  - Playwright MCP
  - Lighthouse
  - AXE-Core
- What to test (Criteria)?
  - Contrast
  - Keyboard
  - Semantics
  - A11Y Tree
  - Alt Text
  - Language
  - Content
  - Responsiveness
  - etc....
- Standards which should be supported
  - WCAG 2.2 AA
  - EN 301549
- Outcome/Report?
  - MD File
  - Jira compliant format

**Storefront Documentation**

- ~Remove "Accessibility Statement" Page~
- Add new Page "AI A11Y Audit"
- Provide docs and example prompts
- Maybe include direct Cursor Prompt links (see: https://cursor.com/docs/agent/tools/browser#accessibility-improvements)

**Automation**

How can this Skill be automated?

- GitHub Workflow
- As Cloud Agent?

**Try out the package https://github.com/Community-Access/accessibility-agents as well. A customized fork would probably also valuable.**

### Acceptance criteria

- [ ] Provide detailed instructions and guidance on PDS Storefront on how to perform an accessibility audit with an AI Agent

**Comment by o-hook** [comment]
BLOCKED: until Storefront deployments are fixed to get preview PR link.
